src/controllers/OrderController.js

```diff
@@ -0,0 +1,13 @@
+import { order } from '../models/store';
+
+class OrderController {
+  setOrder(request, response) {
+    request.body.datatime = new Date();
+    request.body.status = 'pending';
```

Comment:
Look into [de-structuring](http://exploringjs.com/es6/ch_destructuring.html)  the request.body object.